docker-proxy-rule-core/build.gradle

```diff
@@ -7,4 +7,9 @@ dependencies {
     testImplementation group: 'junit', name: 'junit'
     testImplementation group: 'org.assertj', name: 'assertj-core'
     testImplementation group: 'org.mockito', name: 'mockito-core'
+    testRuntimeOnly group: 'org.mockito', name: 'mockito-inline'
+}
+
+moduleJvmArgs {
+    opens 'java.base/java.net'
```

Comment:
This allows the module to work on 16+ (assuming baseline plugins to set the args)